website/pages/docs/reference/cli/_meta.json

```diff
@@ -1,4 +1,5 @@
 {
   "cloudquery": "cloudquery",
-  "cloudquery_sync": "cloudquery sync"
+  "cloudquery_sync": "cloudquery sync",
+  "cloudquery_migrate": "cloudquery migrate"
```

Comment:
Nice, maybe we can add something to our CLI Makefile https://github.com/cloudquery/cloudquery/blob/75162dd372e17b90a387a552d63f61d4c177131b/cli/Makefile#L13 to generate the `_meta.json` file (unrelated to this PR)